src/main/java/baseball/operator/BaseballGame.java

```diff
@@ -0,0 +1,43 @@
+package baseball.operator;
+
+import baseball.View;
+import baseball.domain.Pitch;
+import baseball.domain.Result;
+import baseball.utility.DataConverter;
+import baseball.service.BaseballService;
+
+import java.util.List;
+
+public class BaseballGame {
+
+    public boolean terminate = false;
+    private final BaseballService baseballService;
+
+    public void inningStart() {
+        View.inputNumber();
+        List<Pitch> inputPitches = DataConverter.readData();
+        Result result = baseballService.evaluate(inputPitches);
+
+        View.showResult(result);
+        analyzeResult(result);
+    }
+
+
+    private void analyzeResult(Result result) {
+        if (result.getStrike() == 3) {
+            terminate = true;
+            View.congratulation();
+            return;
+        }
+        terminate = false;
+    }
```

Comment:
```suggestion
    private void analyzeResult(Result result) {
        if (result.getStrike() == 3) {
            terminate = true;
            View.congratulation();
        }
    }
```
이렇게 해도 코드 진행에는 문제가 없어보이는데 어떤가요..??

Reply:
> 이렇게 해도 코드 진행에는 문제가 없어보이는데 어떤가요..??

이러면 게임 종료 후 게임을 다시 시작하게 되었을 때 Controller 에서 문제가 발생할것같습니당..!